### PR TITLE
Finally I did it.

### PR DESCRIPTION
**Pull request by Josofico**
# Take me a while

This exercises were so interesting and take a while to understand, I'm waiting for more in week 4.

### CODE REVIEW

File: student-cvs/Kick-Off-2479/package-lock.json
Comment (Lealdos) [inline student-cvs/Kick-Off-2479/package-lock.json:1]: # Delete file
This file is not necessary on the remote repository, add it to your git ignore, please.

File: student-forms/Kick-Off-2479/bootstrap-setup/package-lock.json
Comment (Lealdos) [inline student-forms/Kick-Off-2479/bootstrap-setup/package-lock.json:1]: # Delete file from  the remote

File: student-forms/Kick-Off-2479/bootstrap-setup/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "start": "vite",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/bootstrap": "^5.2.6",
+    "sass": "^1.60.0",
```
Comment (Lealdos) [inline student-forms/Kick-Off-2479/bootstrap-setup/package.json:15]: # Correct sass location in the file
Sass suppose to be in dependence not in devdependence

**Review by Lealdos — CHANGES_REQUESTED** [review]
## You need to delete some files please

File: student-cvs/Kick-Off-2479/package-lock.json
Comment (Lmendev) [reply student-cvs/Kick-Off-2479/package-lock.json:1]: No. 

[Está bien que lo agregue.](https://www.ictshore.com/javascript/package-lock-json-source-control/)

**Review by Lmendev — APPROVED** [review]

**Review by Lealdos — APPROVED** [review]

**Pull request merged**